README.md

```diff
@@ -1,3 +1,26 @@
 # Compatibility layer
 
 The compatibility layer of the EESSI project uses [Gentoo Prefix](https://wiki.gentoo.org/wiki/Project:Prefix)
+
+## Installation of Gentoo Prefix
+
+- Build a Singularity container with the provided definition file:
+```
+sudo singularity build prefix.simg singularity-prefix.def
+```
+
+- Run the bootstrap script inside the container:
+```
+singularity exec ./centos8.simg ./bootstrap-prefix.sh
+```
+
+- Answer the simple questions, e.g. about the installation path, and wait 
+for a couple of hours for the script to complete.
+
```

Comment:
The installation path can also be set in the environment as $EPREFIX

Reply:
Haven't tried this myself (does that also skip the other questions about wishing the script luck?), but added a note about it.

Reply:
It does not skip the questions, but saves time typing

Reply:
And it avoids accidentally installing to a different location... :)

+1 for defining `$EPREFIX` up front

I also had to do this, because some `/etc/profile.d` scripts in the system I'm playing on define `$LD_LIBRARY_PATH` & co:

```shell
export SHELL="/bin/bash --norc --noprofile
```

The Prefix bootstrap script gives up when it sees things like `$LD_LIBRARY_PATH` and `$CPATH` defined in a freshly created shell.